tools/sigclient/pkg/query/query.go

```diff
@@ -909,3 +910,84 @@ func RunQueryFromFileAndOutputResponseTimes(dest string, filepath string, queryR
 
 	log.Infof("RunQueryFromFileAndOutputResponseTimes: Query results written to CSV file: %v", queryResultFile)
 }
+
+func RunBenchmarkUUIDQuery(uuidList []string, batchID int, wg *sync.WaitGroup, semaphore chan struct{}) {
+
+	defer wg.Done()
+	defer func() { <-semaphore }()
+	for _, uuid := range uuidList {
+
+		data := map[string]interface{}{
+			"state":         "query",
+			"searchText":    "ident=" + uuid,
+			"startEpoch":    "now-1d",
+			"endEpoch":      "now",
+			"indexName":     "*",
+			"queryLanguage": "Splunk QL",
+		}
+		evaluationType := "total"
+		relation := "gt"
+		expectedValue := "0"
```

Comment:
Ok, then maybe this should be equal to the value of `p`. My concern is that with `> 0`, some bugs could potentially slip in unnoticed that would have been caught if we tested for equality

Reply:
For 100 queries , p=3 , the end for first batch will be adjusted to add the remainder of 1 and  will have one query with unique ident column not present in other batches. So checking for >=p fails .

Reply:
Ok. Do you think you can get it to work with equality, or should we keep this pretty relaxed constraint of `> 0`?

Reply:
we can add a check if processCount==1 then we can use `eq 1`  else >0. This also assumes we start fresh with no data otherwise subsequent ingestion will also have same uuid which will increase the hits count

Reply:
for this gentype subsequent ingestion should not cause any issue. This should pick a random seed so that the results are not dependent on whether previously data was present or not. If we want to pick the same seed, then a flag can be passed